corehq/apps/ota/views.py

```diff
@@ -333,9 +333,10 @@ def get_next_id(request, domain):
 # Note: this endpoint does not require authentication
 @location_safe
 @require_GET
+@toggles.MOBILE_RECOVERY_MEASURES.required_decorator()
```

Comment:
@amstone326 I made the feature flag also control access to the recovery endpoint.  This isn't strictly necessary, but I thought it'd be prudent, so in the future we can see exactly which domains might be using this.  Let me know if you see any issues with it (for instance, would we want to disable the profile change and/or this build in the future?  What would that look like?)